ludwig/datasets/configs/poker_hand.yaml

```diff
@@ -28,6 +16,29 @@ description: |
   order of cards is important, which is why there are 480 possible
   Royal Flush hands as compared to 4.
   https://archive.ics.uci.edu/ml/datasets/Poker+Hand
+columns:
+  - name: S1
+    type: category
+  - name: C1
+    type: category
+  - name: S2
+    type: category
+  - name: C2
+    type: category
+  - name: S3
+    type: category
+  - name: C3
+    type: category
+  - name: S4
+    type: category
+  - name: C4
+    type: category
+  - name: S5
+    type: category
+  - name: C5
+    type: category
+  - name: hand
+    type: category
```

Comment:
From internal benchmarks, I'd make all of these number types since the model seems to perform better, but leave the output as category since that is correct!

Reply:
Sounds good, will change!